docs/docs/Deployment/deployment-prod-best-practices.mdx

```diff
@@ -57,3 +57,223 @@ This separation is designed to enhance security, optimize resource allocation, a
 * **Resource allocation**
```

Comment:
I can't comment on it, but the heading on line 49 should be H3 since it continues the "Environment isolation" section.